views/mail/order/shipped.htm

```diff
@@ -1,13 +1,13 @@
-subject = "Your order has been shipped"
+subject = "{{ 'offline.mall::mail.order.shipped.subject' | trans() }}"
 description = "Sent when tracking information was added"
 ==
 
-Hi {{ order.customer.firstname }}
+{{ 'offline.mall::mail.common.hello' | trans() }} {{ order.customer.firstname }}
```

Comment:
Try to use placeholders in these cases as well! There might be locales where the firstname won't be the last word of the sentence but somewhere in the middle.

```suggestion
{{ 'offline.mall::mail.common.hello' | trans({firstname: order.customer.firstname}) }}
```